### ISSUE TITLE
Expose OM inputs in HA

### ISSUE THREAD

**Issue by geiregjo** [openmotics/home-assistant#30]
I'm want to detect in Home Assistant when a OpenMotics input was triggered. I'd like to use this to control things that don't integrate with OpenMotics, like Somfy IO powered shutters.

**Comment by woutercoppens** [comment]
I have added some code to expose inputs, but until we get websockets working there will be a delay in updating the status.
Secondly I have to figure out what Home-Assistant component can be use to expose the inputs.

**Comment by geiregjo** [comment]
I believe you need to use a [Binary Sensor](https://www.home-assistant.io/integrations/binary_sensor/)?

**Comment by kneirinck** [comment]
Yeah I think so as well, other option is the Switch entity but that implies you can activate it as well which isn't the case with an input for openmotics I think?
But as stated by @woutercoppens I don't think it'll be very useful without the websockets implementation.
Because that would mean you need to press it exactly when the periodic scan happens of the state.
If the input is triggered any other time it won't be picked up :/

**Comment by woutercoppens** [comment]
Anyone willing to help: https://github.com/woutercoppens/pyhaopenmotics-v2/tree/websockets

I cannot get the websockets working, nether on the local gateway nether on the cloud. Have a look at the examples.
There are 2 functions: connect & connect2. I tried it via aiohttp or via websockets.
